Learn-JSON-Fetch-async-await-building-fcc-author-page/index.html

```diff
@@ -0,0 +1,18 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>freeCodeCamp Author page</title>
```

Comment:
Very minor suggestion:

```suggestion
    <title>freeCodeCamp News Author Page</title>
```

But the current `title` works, too.